spec/system/user_reset_progress_spec.rb

```diff
@@ -21,7 +21,7 @@
 
   it 'resets progress' do
     expect(user.lesson_completions.count).to eq(2)
-    within(find('.skills')) do
+    within(find_by_id('skills')) do
```

Comment:
With the tests id data attribute selector in place this and the other instances of `find_by_id` would change to:
```suggestion
    within(find(:test_id, 'skills')) do
```